### ISSUE TITLE
Feature Request: Cumstomizable severity level for flying text and play beep sound for TTS

### ISSUE THREAD

**Issue by fly1n** [xpdota/event-trigger#79]
Basically two features that cactbot had and were useful.

**Comment by xpdota** [comment]
I recently released the ability to customize callout colors a bit, if that's what you're looking for:

![image](https://user-images.githubusercontent.com/14287379/174393764-18e9f462-2f17-4038-8d84-82be08cfef18.png)

As for sound files, I'll have to think of a good way of adding it.

**Comment by fly1n** [comment]
Yes. The customizable callout color can do it, but I think most of the time 3 levels (info, warning, error) work as the preset.
